client/jetpack-cloud/sections/partner-portal/types.ts

```diff
@@ -34,3 +34,13 @@ export interface AssignLicenceProps {
 	selectedSite?: SiteDetails | null;
 	suggestedProduct?: string;
 }
+
+export interface LicenceAction {
```

Comment:
Non-critical:

It's probably a good idea for us to settle on one uniform spelling of license/licence. I know both are valid depending on the dialect of English being used, but I see both spellings in this PR.